### ISSUE TITLE
How to configure pyls-mypy - python type checker ?

### ISSUE THREAD

**Issue by yml** [neoclide/coc.nvim#467]
**Is your feature request related to a problem? Please describe.**

I pip installed `pyls-mypy` wich is a an optional dependency of `pyls` that offers typchecking for pyhton 3. To a lot of my surprise it seems to work out of the box with `coc-pyls` to some extend:
```
 project/apps/base_photocontest/models/abstracts.py:6:2 Error [mypy] No library stub file for module 'django.core.files.storage'
project/apps/base_photocontest/models/abstracts.py:7:2 Error [mypy] No library stub file for module 'django.core.validators'
```

The issue is that I need to gives pyls some hint on where to find the [type library for django](https://github.com/machinalis/mypy-django). As far as I understand this is done by setting an environement variable called `MYPYPATH`

```
export MYPYPATH=/home/dmoisset/mypy-django/
```

Another issue is that I would like to activate and deactivate the mypy plugin in `:CocConfig` with something like this:
```
 "pyls.plugins.mypy.enabled": false,
```
In a nutshell am a bit clue less on how to  configure pyls beyond what the already very nice autocomplete expose in `:CocConfig`

----
Thank you for the amazing work you are doing with coc.nvim

**Comment by chemzqm** [comment]
That should an option provided by pyls, but I don't know if it has it or not.

**Comment by chemzqm** [comment]
Cinsider try out https://github.com/neoclide/coc-python, I can't help with issues of pyls.
